BitwardenShared/UI/Auth/ProfileSwitcher/ProfileSwitcherToolbarView.swift

```diff
@@ -17,18 +17,14 @@ struct ProfileSwitcherToolbarView: View {
         AsyncButton {
             await store.perform(.requestedProfileSwitcher(visible: !store.state.isVisible))
         } label: {
-            HStack {
-                profileSwitcherIcon(
-                    color: store.state.showPlaceholderToolbarIcon
-                        ? nil : store.state.activeAccountProfile?.color,
-                    initials: store.state.showPlaceholderToolbarIcon
-                        ? nil : store.state.activeAccountProfile?.userInitials,
-                    textColor: store.state.showPlaceholderToolbarIcon
-                        ? nil : store.state.activeAccountProfile?.profileIconTextColor
-                )
-                Spacer()
-            }
-            .frame(minWidth: 50)
+            profileSwitcherIcon(
+                color: store.state.showPlaceholderToolbarIcon
+                ? nil : store.state.activeAccountProfile?.color,
+                initials: store.state.showPlaceholderToolbarIcon
+                ? nil : store.state.activeAccountProfile?.userInitials,
+                textColor: store.state.showPlaceholderToolbarIcon
+                ? nil : store.state.activeAccountProfile?.profileIconTextColor
```

Comment:
⛏️ Can you fix the indentation warnings here?

```suggestion
                color: store.state.showPlaceholderToolbarIcon
                    ? nil : store.state.activeAccountProfile?.color,
                initials: store.state.showPlaceholderToolbarIcon
                    ? nil : store.state.activeAccountProfile?.userInitials,
                textColor: store.state.showPlaceholderToolbarIcon
                    ? nil : store.state.activeAccountProfile?.profileIconTextColor
```

Reply:
thanks!
just fixed.